packages/rest/test/unit/coercion/paramStringToBoolean.unit.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {testCoercion} from './utils';
+
+describe('coerce param from string to boolean', () => {
+  it("value 'false' is coerced to false", async () => {
+    const caller = new Error().stack!.split(/\n/)[1];
```

Comment:
Can you move this line to `testCoercion` please, so that we don't have to duplicate it in so many places?

In fact, since we are invoking `testCoercion` from inside a test function, I don't think we need this `caller` line at all, because the stack trace for any error thrown from inside `testCoercion` should already contain a stack frame (a stack line) pointing to the exact test.

I was proposing to use `caller` computed via `new Error().stack` for the design where both the test name and test function is created by the helper:

```js
it(...testCoercion({type: 'boolean'}, 'true', true));
```

Since your patch is using a different approach to parameterize tests, I think you should simply remove `caller` variable from all tests and the helper arguments.

Thoughts?